soc/silabs/silabs_siwx91x/Kconfig.defconfig

```diff
@@ -13,6 +13,11 @@ configdefault SYS_CLOCK_TICKS_PER_SEC
 	default 128 if !TICKLESS_KERNEL && SILABS_SLEEPTIMER_TIMER
 	default 1024 if SILABS_SLEEPTIMER_TIMER
 
+if PM
```

Comment:
This could be moved to `default y if PM` on the definition of `SOC_SIWX91X_PM_BACKEND_PMGR` instead of being a defconfig override.